packages/typespec-test/test/openai_generic/spec/images/operations.tsp

```diff
@@ -11,13 +11,6 @@ namespace OpenAI;
 
 @route("/images")
 interface Images {
-  @route("generations")
-  @post
-  @operationId("createImage")
-  @tag("OpenAI")
-  @summary("Creates an image given a prompt")
-  create(@body image: CreateImageRequest): ImagesResponse | ErrorResponse;
```

Comment:
This is a case in which normal operation will include HttpPart model. I just removed this case considering this is not finalized yet: https://github.com/microsoft/typespec/issues/6530.